migrations/src/migrations/migration_0003_permissions.rs

```diff
@@ -0,0 +1,38 @@
+use sqlx_simple_migrator::{migration_name, Migration};
+
+pub fn migration() -> Migration {
+    Migration::new(migration_name!())
+        .with_up(
+            r#"
+            CREATE TABLE permission_groups (
+                id SERIAL PRIMARY KEY,
+                name TEXT NOT NULL,
+                created_at TIMESTAMPTZ NOT NULL DEFAULT now()
+            )
+        "#,
+        )
+        .with_down("DROP TABLE IF EXISTS permission_groups")
+        .with_up(
+            r#"
+            CREATE TABLE permission_group_statements (
+                id SERIAL PRIMARY KEY,
+                permission_group_id INT NOT NULL REFERENCES permission_groups(id),
+                service TEXT NOT NULL,
+                permission TEXT NULL,
+                created_at TIMESTAMPTZ NOT NULL DEFAULT now()
```

Comment:
Add a unique constraint for group, service, permission.

Add an index for permission_group_id